pepper-apis/src/main/resources/org/broadinstitute/ddp/db/dao/StudyGovernanceDao.sql.stg

```diff
@@ -36,3 +41,35 @@ queryPolicyByStudyGuid() ::= <<
 where study.guid = :studyGuid
 order by aom.execution_order asc
 >>
+
+select_all_age_up_candidates() ::= <<
+select auc.age_up_candidate_id,
+       study.umbrella_study_id as study_id,
+       study.guid              as study_guid,
+       participant.user_id as participant_user_id,
+       participant.guid    as participant_user_guid,
+       profile.birth_date,
+       auc.initiated_preparation,
+       (select et.enrollment_status_type_code
+          from user_study_enrollment as en
+          join enrollment_status_type as et on et.enrollment_status_type_id = en.enrollment_status_type_id
+         where en.study_id = study.umbrella_study_id
+           and en.user_id = participant.user_id
+           and valid_to is null
+       ) as enrollment_status
+  from age_up_candidate as auc
+  join umbrella_study as study on study.umbrella_study_id = auc.study_id
+  join user as participant on participant.user_id = auc.participant_user_id
+  left join user_profile as profile on profile.user_id = participant.user_id
```

Comment:
I thought we were always creating `user_profile` during registration--this seems to be the case in `UserRegistrationRoute`.  Should we nix the left join?

Reply:
I'll still need to join to `user_profile` to grab the `birth_date`. But if we did an inner `join` and somehow user doesn't have profile, then we wouldn't know because they will be excluded from the result set. I think I'll keep the `left join` here -- the log in `AgeUpService` will give us some indication of a missing profile.